regression/cbmc-java/lambda2/test_no_crash_ecj.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+StaticMethodRef.class
+--function StaticMethodRef.test2
```

Comment:
Why this method is tested instead of `Smr`? Wouldn't `Smr` do the same job, with lambdas being ignored but parsed?

Reply:
I agree and the test passes setting to the the `Smr` so will change to that but @mgudemann when you're back if you could comment on whether this was the right thing to do.